packages/ozone-api/ozone-api-upload/src/ozone-api-upload.ts

```diff
@@ -1,7 +1,7 @@
 import { OzoneConfig } from 'ozone-config'
 import { OzoneAPIRequest } from 'ozone-api-request'
 import { getDefaultClient } from 'ozone-default-client'
-import { UUID } from 'ozone-type'
+import { TaskHandlerOption } from '../../../ozone-typescript-client/src/taskClient/taskClient'
```

Comment:
`import { TaskHandlerOption } from 'ozone-typescript-client/src/taskClient/taskClient'`

Reply:
you may need to add it as a dependency